### PR TITLE
Clone page

### PR DESCRIPTION
**Pull request by Maryjanee**
Completed Youtube clone project

### CODE REVIEW

**Review by sumancrest0001 — COMMENTED** [review]
**Status: Changes required
Hello TEAM :wave:** 
You have done a nice job to create an almost similar layout, the readme file is also looking good. But there are some minor issues that need to be addressed before moving ahead. let's fix them.
[Required] The YouTube button on the left should align with other contents. Currently, they are not aligned properly. :point_down: 
Screenshot from 2020-06-04 07-41-49
It should be :point_down: 
Screenshot from 2020-06-04 07-47-37

[Optional] You can change the logo also to give it similar looks.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+                  <li class="icons">
+                    <img src="images/like.svg" alt="like" /> 422
+                  </li>
+                  <li class="icons">
+                    <img src="images/dislike.svg" alt="dislike" />18
+                  </li>
+                  <li class="icons">
+                    <img src="images/share.svg" alt="share" /> share
+                  </li>
+                </ul>
```
Comment (sumancrest0001) [inline index.html]: [Required] Share icon should be a button but it is an item only in your layout. [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/video_share_requirement.png) of the requirement.

[Required] Thumbnail images of other videos are not showing.I am using [githack](https://raw.githack.com/) to check the project.  Have a look :point_down: 
Screenshot from 2020-06-04 07-37-30

**Review by sumancrest0001 — CHANGES_REQUESTED** [review]
**Kindly make the requested changes and request another review.**

If you have any queries regarding the review or need support, message me on slack **Suman Shrestha**. I am happy to help you.
**Happy coding !!!**

**Comment by rahalrazika** [comment]
All changes requested have been done and the readme file has been updated with a live demo of the project.

**Comment by elmejdki** [comment]
### Pull request under review by Zakariae El Mejdki.

File: README.md [hunk 2]
```diff
@@ -1 +1,40 @@
...
+
+## Authors
+
+👤 **Maryjane**
+
+- Github: [@maryjane](https://github.com/maryjanee)
+
+👤 **Razika**
+
+- Github: [@rahalrazika](https://github.com/rahalrazika)
```
Comment (elmejdki) [inline README.md]: [OPTIONAL] if it is possible add also your Linkedin and Twitter profile links here.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,194 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+/* Basic resets */
```
Comment (elmejdki) [inline style.css]: kindly remove all the comments from all the files:
image

File: style.css [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+
+#show {
+  color: grey;
+  display: block;
+  margin-top: 1rem;
+  text-transform: uppercase;
+  padding-left: 3.2rem;
+}
+
+/* aside styles */
```
Comment (elmejdki) [inline style.css]: remove this comment also.

File: style.css [hunk 5]
```diff
@@ -0,0 +1,194 @@
...
+
+/*Main Styles */
+#main-container {
+  padding: 2rem 6rem;
+
+  display: flex;
+  justify-content: space-between;
+}
+
+/* Video-container */
```
Comment (elmejdki) [inline style.css]: also this one.

File: style.css [hunk 6]
```diff
@@ -0,0 +1,194 @@
...
+
+.icons {
+  margin-right: 1.5rem;
+  width: 24px;
+}
+#avatar {
+  margin-right: 0;
+}
+
+/*Main Styles */
```
Comment (elmejdki) [inline style.css]: and this one.

File: style.css [hunk 7]
```diff
@@ -0,0 +1,194 @@
...
+nav {
+  width: 100%;
+  padding: 1rem;
+}
+
+#logo {
+  width: 8rem;
+}
+
+/* Form Styles */
```
Comment (elmejdki) [inline style.css]: another comment to be deleted.

File: style.css [hunk 8]
```diff
@@ -0,0 +1,194 @@
...
+
+/* Basic resets */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+}
+
+/* General styles */
```
Comment (elmejdki) [inline style.css]: remove this comment as suggested above.

File: style.css [hunk 9]
```diff
@@ -0,0 +1,194 @@
...
+h4,
+h5,
+h6 {
+  margin: 1rem 0;
+}
+button {
+  cursor: pointer;
+}
+
+/* Navigation Styles */
```
Comment (elmejdki) [inline style.css]: and this one.

**Review by elmejdki — CHANGES_REQUESTED** [review]
# Great job so far team :woman_technologist: :tophat: 
### status: require changes

- Add a meaningful title and summary on your pull request, you can use on the title *Completed Youtube clone project*, and on the summary try to add some bullet point of what you have accomplished, 

- use this link [HTLM validator](https://validator.w3.org/#validate_by_uri) to validate your HTML code, while validating it I have found the following errors:

image
- To fix this error please remove any HTML styling attributes and use CSS instead. 

image
- to fix this one also you need to add a heading and then use a display none in your CSS, to make it invisible.

### please make the required changes before requesting another pull request
_if you have any question be free to contact me on slack you will find me as **Zakariae El Mejdki**_

**Comment by Maryjanee** [comment]
- Added navigation bar
- Embedded Video and videos on webpage
- Added sidebar with recommended images.
- Removed the comments in the css file
- Validated index File with HTML validator and fixed all resulting issues.

File: index.html [hunk 10]
```diff
@@ -0,0 +1,173 @@
...
+      <aside>
+        <div id="aside-title">
+          <p>Up next</p>
+          <p class="icons d-flex">
+            Autoplay<img class="icons" src="images/switch.svg" alt="switch" />
+          </p>
+        </div>
+        <div class="thumbnail-container">
+          <div id="first-video" class="thumbnail d-flex">
+            <img src="images/1.png" alt="thumbnail_1" />
```
Comment (HADDADSOHAIB) [inline index.html:108]: I see that the name of your file in the image folder is 1.PNG, that could be the issue why the images are not showing

**Review by HADDADSOHAIB — CHANGES_REQUESTED** [review]
> ### Pull Request under the review by Sohaib Haddad. (TSE)
### Change Required :recycle: 

### Hello @Maryjanee and @rahalrazika     :handshake: 

### You did wonderful job :clap:  your project is looking good :medal_sports: and look like the original. 

### You implemented the changes required by the previous TSE, good job, however I found some issues while checking your project latest version using https://raw.githack.com/, (the live demo provided in the readme is ok), there are some changes to get the project approved :point_down:
- the thumbnails are not showing, check the comments for suggesting.
Screenshot from 2020-06-05 13-06-44
 ### Please fix these issues and asked for another code review.

**Comment by BrittanyBlake** [comment]
### Pull Request Under Review by Brittany Blake (TSE)

**Review by BrittanyBlake — APPROVED** [review]
### Status: Approved ✅
****
### Hi @Maryjanee and @rahalrazika 👋 👋 
You have done a great job implementing the changes requested by the previous TSEs! 👍 
### Congratulations 🏅 👏 
Go ahead and merge this pull request and move on to the next project 💪 

HAPPY CODING! 👩🏾‍💻

**Pull request merged**